releasenotes/notes/runtime-security-config-file-handling-a9508fda817b9b30.yaml

```diff
@@ -0,0 +1,6 @@
+---
+enhancements:
+  - |
+    The security agent will fail quickly if the first of either the default, or if they exist, user-provided override config files are inaccessible.
+    The default config files are still datadog.yaml and security-agent.yaml, in that order.
+    Previously, if the security agent was started with at least one accessible config file, no matter the order, the agent would still be able to start.
```

Comment:
```suggestion
    Previously, if the security agent was started with at least one accessible config file, no matter what order the files were loaded, the Datadog Agent would still be able to start.
```

Reply:
I'm assuming the agent mentioned here is the Datadog Agent, let me know if that's wrong.

Reply:
It's the security agent. The new sentence is 
```
    Previously, if the security agent was started with at least one accessible config file, no matter what order, the security agent would still be able to start.
```

Reply:
That sentence works, just added a small edit, so it's "no matter in what order". I updated my suggested change.

Reply:
implementing!